CMakeLists.txt

```diff
@@ -6,9 +6,11 @@ if(NOT CMAKE_CXX_STANDARD)
   set(CMAKE_CXX_STANDARD 14)
 endif()
 
-# Disable Wredundant-decls warnings since rosidl generates redundant function declarations
+# Disable Wredundant-decls warnings since rosidl generates redundant function
+#  declarations
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Wno-redundant-decls")
+  set(CMAKE_CXX_FLAGS 
+    "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Wno-redundant-decls")
```

Comment:
Which linter enforces this? `ament_lint_cmake`?

Reply:
yes, it enforces a 80 character line limit in cmake files